test/MasterViewController.m

```diff
@@ -46,15 +56,33 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
 {
     UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"myCell" forIndexPath:indexPath];
-    Employee *employee = self.org.employees[indexPath.row];
-    cell.textLabel.text = employee.fullName;
+   // NSSortDescriptor *firstNameDescriptor = [[NSSortDescriptor alloc] initWithKey:@"firstName" ascending:true];
+   // self.sortedArrayInCell = [self.org.employees sortedArrayUsingDescriptors:@[firstNameDescriptor]];
+    
+    Employee *employeeToCell = self.sortedEmployees[indexPath.row];
+    [cell.textLabel setText:[NSString stringWithFormat:@"%@", [employeeToCell valueForKey:@"fullName"]]];
```

Comment:
Why are you using KVC, if you know the class of the object?
Why are you using stringWithFormat, if you already have the composed string.